src/main/flight/pid.h

```diff
@@ -64,11 +64,16 @@ typedef struct pidProfile_s {
     uint8_t  D8[PID_ITEM_COUNT];
     uint8_t  pidController;
     uint16_t yaw_p_limit;                   // set P term limit (fixed value was 300)
-    uint16_t dterm_lpf;                     // dterm filtering
-    uint16_t yaw_lpf;                       // additional yaw filter when yaw axis too noisy
+    uint16_t dterm_lpf_hz;                  // Delta Filter in hz
+    uint16_t yaw_lpf_hz;                    // additional yaw filter when yaw axis too noisy
     uint8_t  deltaMethod;
+    uint8_t dterm_filter_type;              // Filter selection for dterm
```

Comment:
oh i see, i DON'T need to change it if i just append the fields.  hmm, for some reason i thought i had to change it...  let me re-order and keep the version the same.